### PR TITLE
cloudstorage(ticdc):  Remove nfs schema file check

### LINKED ISSUE #10137: ticdc panic because of no table schema file found in an non-empty meta path
```
[2023/11/22 08:46:15.708 +08:00] [PANIC] [path.go:224] ["no table schema file found in an non-empty meta path"] [versionedTableName="{\"TableNameWithPhysicTableID\":{\"db-name\":\"xxxx\",\"tbl-name\":\"xxxx\",\"tbl-id\":324608,\"is-partition\":false},\"TableInfoVersion\":445795210187767818}"] [checksum=3785739126] [stack="[github.com/pingcap/tiflow/pkg/sink/cloudstorage.(*FilePathGenerator).CheckOrWriteSchema\n\tgithub.com/pingcap/tiflow/pkg/sink/cloudstorage/path.go:224\ngithub.com/pingcap/tiflow/cdc/sinkv2/eventsink/cloudstorage.(*dmlWorker).flushMessages\n\tgithub.com/pingcap/tiflow/cdc/sinkv2/eventsink/cloudstorage/dml_worker.go:162\ngithub.com/pingcap/tiflow/cdc/sinkv2/eventsink/cloudstorage.(*dmlWorker).run.func1\n\tgithub.com/pingcap/tiflow/cdc/sinkv2/eventsink/cloudstorage/dml_worker.go:135\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\tgolang.org/x/sync@v0.1.0/errgroup/errgroup.go:75](http://github.com/pingcap/tiflow/pkg/sink/cloudstorage.(*FilePathGenerator).CheckOrWriteSchema/n/tgithub.com/pingcap/tiflow/pkg/sink/cloudstorage/path.go:224/ngithub.com/pingcap/tiflow/cdc/sinkv2/eventsink/cloudstorage.(*dmlWorker).flushMessages/n/tgithub.com/pingcap/tiflow/cdc/sinkv2/eventsink/cloudstorage/dml_worker.go:162/ngithub.com/pingcap/tiflow/cdc/sinkv2/eventsink/cloudstorage.(*dmlWorker).run.func1/n/tgithub.com/pingcap/tiflow/cdc/sinkv2/eventsink/cloudstorage/dml_worker.go:135/ngolang.org/x/sync/errgroup.(*Group).Go.func1/n/tgolang.org/x/sync@v0.1.0/errgroup/errgroup.go:75)"]
```

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: close #10137

### What is changed and how it works?
this pr remove the schema file checking, because some cdc will generate a tmp file and then rename it, but these two operation is not atomic,so if ticdc is stopped after generating the tmp file, cdc has no chance to rename the tmp file,  cdc will panic again and again because of the checking logic

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by hicqu — APPROVED** [review]

File: pkg/sink/cloudstorage/path.go [hunk 1]
```diff
@@ -219,19 +219,14 @@ func (f *FilePathGenerator) CheckOrWriteSchema(
 	}
 
 	// Case 2: the table meta path is not empty.
-	if schemaFileCnt != 0 {
-		if lastVersion == 0 {
-			log.Panic("no table schema file found in an non-empty meta path",
-				zap.Any("versionedTableName", table),
-				zap.Uint32("checksum", checksum))
-		}
+	if schemaFileCnt != 0 && lastVersion != 0 {
```
Comment (nongfushanquan) [inline pkg/sink/cloudstorage/path.go:222]: A log could be recorded here

Comment (sdojjy) [reply pkg/sink/cloudstorage/path.go:222]: done

**Review by nongfushanquan — APPROVED** [review]

**Pull request merged**